web/src/message_edit_history.ts

```diff
@@ -50,81 +84,86 @@ export function fetch_and_render_message_history(message) {
+                        prev_stream = $t({defaultMessage: "Unknown stream"});
                     } else {
-                        item.prev_stream = sub_store.maybe_get_stream_name(msg.prev_stream);
+                        prev_stream = sub_store.maybe_get_stream_name(msg.prev_stream);
                     }
                     if (prev_stream_item !== null) {
                         prev_stream_item.new_stream = sub_store.maybe_get_stream_name(
                             msg.prev_stream,
                         );
                     }
-                    prev_stream_item = item;
                 } else if (msg.prev_topic) {
-                    item.edited_by_notice = $t(
-                        {defaultMessage: "Moved by {full_name}"},
-                        {full_name},
-                    );
-                    item.topic_edited = true;
-                    item.prev_topic = msg.prev_topic;
-                    item.new_topic = msg.topic;
+                    edited_by_notice = $t({defaultMessage: "Moved by {full_name}"}, {full_name});
+                    topic_edited = true;
+                    prev_topic = msg.prev_topic;
+                    new_topic = msg.topic;
                 } else if (msg.prev_stream) {
                     const sub = sub_store.get(msg.prev_stream);
-                    item.edited_by_notice = $t(
-                        {defaultMessage: "Moved by {full_name}"},
-                        {full_name},
-                    );
-                    item.stream_changed = true;
+                    edited_by_notice = $t({defaultMessage: "Moved by {full_name}"}, {full_name});
+                    stream_changed = true;
                     if (!sub) {
-                        item.prev_stream = $t({defaultMessage: "Unknown stream"});
+                        prev_stream = $t({defaultMessage: "Unknown stream"});
                     } else {
-                        item.prev_stream = sub_store.maybe_get_stream_name(msg.prev_stream);
+                        prev_stream = sub_store.maybe_get_stream_name(msg.prev_stream);
                     }
                     if (prev_stream_item !== null) {
                         prev_stream_item.new_stream = sub_store.maybe_get_stream_name(
                             msg.prev_stream,
                         );
                     }
-                    prev_stream_item = item;
                 } else {
                     // just a content edit
-                    item.edited_by_notice = $t(
-                        {defaultMessage: "Edited by {full_name}"},
-                        {full_name},
-                    );
-                    item.body_to_render = msg.content_html_diff;
+                    edited_by_notice = $t({defaultMessage: "Edited by {full_name}"}, {full_name});
+                    body_to_render = msg.content_html_diff;
+                }
+
+                const item = {
```

Comment:
`item: ContentEditHistoryItem` would be nice for readability.

Reply:
Sure, added!